app/controllers/api/v1x0/mixins/rbac_mixin.rb

```diff
@@ -39,9 +38,8 @@ def permission_array_check(permissions)
         private
 
         def permission_format_check(permissions)
-          return unless RBAC::Access.enabled?
```

Comment:
@mkanoor is this change ( and on line 30 ) part of fixing this bug?

Reply:
@syncrou Yes, we should not be doing a RBAC Access check when we are validating the parameters.